src/rpc/misc.cpp

```diff
@@ -5,15 +5,18 @@
 
 #include <httpserver.h>
 #include <key_io.h>
+#include <node/context.h>
 #include <outputtype.h>
 #include <rpc/blockchain.h>
 #include <rpc/server.h>
 #include <rpc/util.h>
 #include <script/descriptor.h>
+#include <scheduler.h>
 #include <util/check.h>
 #include <util/strencodings.h>
 #include <util/system.h>
 #include <util/validation.h>
+#include <init.h>
```

Comment:
58c4c1943e082060f38054ed4738478635ab9c31

Why is this included?

Reply:
good catch. was left over from a previous iteration